### DISCUSSION TITLE
Error in truffle npm ERR! errno -4058

### DISCUSSION THREAD

**Discussion by emiljoy** [ConsenSys-archive/truffle#4512] [General Q&A]
Hello, I have a problem installing that truffle section to later export my contracts to the tesnet.

Somebody could help me?

npm install --save-dev truffle ganache-cli @truffle/hdwallet-provider
npm ERR! code ENOENT
npm ERR! syscall spawn git
npm ERR! path git
npm ERR! errno -4058
npm ERR! enoent An unknown git error occurred
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! A complete log of this run can be found in:
npm ERR! C:\Users..\AppData\Local\npm-cache_logs\2021-12-08T23_04_30_003Z-debug-0.log

**Comment by apdan510** [comment]
let me know if you are still having issues here. I can assist with this.
